specification/datamigration/resource-manager/Microsoft.DataMigration/stable/2021-06-30/examples/ServiceTasks_Get.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "subscriptionId": "fc04246f-04c5-437e-ac5e-206a19e7193f",
+    "groupName": "DmsSdkRg",
+    "serviceName": "DmsSdkService",
+    "taskName": "DmsSdkTask",
+    "api-version": "2021-06-30"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "input": {
+            "serverVersion": "NA"
+          },
+          "taskType": "Service.Check.OCI",
+          "state": "Queued"
+        },
+        "etag": "0vPYxzfnDaDH9yhOJAnqTyTRpa09Kb7pm+LEukDBbw8=",
+        "id": "/subscriptions/fc04246f-04c5-437e-ac5e-206a19e7193f/resourceGroups/DmsSdkRg/providers/Microsoft.DataMigration/services/DmsSdkService/serviceTasks/DmsSdkTask",
+        "name": "DmsSdkTask",
+        "type": "Microsoft.DataMigration/services/serviceTasks"
+      }
+    }
```

Comment:
This one was for oracle driver right? Now that we do not support that scenario, can we remove it? or we just keep it? I am not sure if it will work though

Reply:
Specially the Create_Or_Update

Reply:
Yeah that was made for the OCI driver update. Russ and I synced on this earlier and we decided to keep all Oracle based tools and perform the cleanup later since we'd be performing a more serious breaking change by removing it.